pkg/insights/metrics.go

```diff
@@ -37,6 +38,15 @@ func init() {
 type InsightsRecommendationCollector struct {
 	activeRecommendations []types.InsightsRecommendation
 	metricName            string
+	clusterID             v1.ClusterID
+}
+
+func (c *InsightsRecommendationCollector) SetClusterID(clusterID v1.ClusterID) {
+	c.clusterID = clusterID
+}
+
+func (c *InsightsRecommendationCollector) GetClusterID() v1.ClusterID {
```

Comment:
nit: See what says [Effective Go](https://go.dev/doc/effective_go#Getters) and yes I know we have these getters with `Get` all over the place,

Reply:
Thank you, good point.